assets/cluster-bootstrap/api-usage.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
Please include a link to the source from which this was obtained.  See some of the other cluster bootstrap resources for examples.

Reply:
The source link has been added at the top of the file.

Reply:
I recommend using a branch link such as https://github.com/openshift/cluster-kube-apiserver-operator/blob/release-4.10/bindata/assets/alerts/api-usage.yaml.  This make it easier to determine the version.